splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceDateFunctions.java

```diff
@@ -113,7 +112,8 @@ public static Time TO_TIME(String source) throws SQLException {
      */
     public static Time TO_TIME(String source, String format) throws SQLException {
         if (source == null) return null;
-        return new Time(parseDateTime(source, format));
+        Date date = TO_DATE(source, format);
+        return new Time(date.getTime()) ;
```

Comment:
TO_DATE() will change the date to local date, as a result, we lose the hour/minute/second info. Won't the Time object created here all be 00:00:00?